generators/client/templates/gulpfile.js

```diff
@@ -30,6 +30,8 @@ var gulp = require('gulp'),
     serve = require('./gulp/serve'),
     util = require('./gulp/utils'),
     gulpIf = require('gulp-if'),
+    inject = require('gulp-inject'),
+    angularFilesort = require('gulp-angular-filesort'),
```

Comment:
Sadly no, you need to add it, because gulp put `<script src="app/app.constants.js"></script>` before `<script src="app/app.module.js"></script>` and is not working.